frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -168,8 +177,18 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
 
         var isReauthJourney = journeyType.equalsIgnoreCase(JourneyType.REAUTHENTICATION.getValue());
 
-        int incorrectPasswordCount =
-                retrieveIncorrectPasswordCount(request.getEmail(), isReauthJourney);
+        int incorrectPasswordCount = 0;
+
+        if (configurationService.isAuthenticationAttemptsServiceEnabled()) {
```

Comment:
Here we should also check if the journey is of type Reauth. This change affects the normal SIGN_IN journey as well.